kits/subscription-audit/apps/.env.example

```
@@ -0,0 +1,4 @@
+AGENTIC_GENERATE_CONTENT="AGENTIC_GENERATE_CONTENT Flow ID"
+LAMATIC_API_URL="LAMATIC_API_URL"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)